scrolling_parameters_object.md

```diff
@@ -0,0 +1,43 @@
+# Overview
+
+For GET requests responses may be [returned in groups](./results_scrolling.md).
+
+## Custom group sizes
+
+It is possible to customize the scrolling parameters for each query. If you want to customize the settings, include the following object in your request:
+
+```JSON
+{
+  "group_size": <integer>
+}
+```
+
+This will return groups of the specified size. Please note the following rules:
+
+- If not specified, the group size will be 25;
+- The group size is limited to a maximum of 25;
+- The group size is limited to a minimum of 1; 
+- If a number outside this range is specified the default will apply; 
+
+### Specifying the custom group sizes
+
+To set the group size parameters include the object in your request:
+
+```JSON
+[
+  {
+    "scrolling": {
+      "group_size": 30
```

Comment:
You use 30 here, but you document the maximum as 25. Should make it consistent.

Reply:
That was there before this I swear!